### DISCUSSION TITLE
Why can't I show promotional items created with promotionCreate mutation in dashboard?

### DISCUSSION THREAD

**Discussion by MohamedEZ-zaalyouy** [saleor/saleor#15530] [General]
Hello,
Why can't I show promotional items created with promotionCreate mutation in dashboard?

i use:

dashboard v3.18.0
core v3.18.9

**Comment by korycins** [comment]
Hey,

Promotions are replacement for Sale. It is still in preview, dashboard by default lists only existing sales there. 
If you want to use `Promotion` you need to enable promotion feature. This can be done from dashboard side. See a screenshot. Then, you will see a modified `Discounts` section which operates on the Promotion feature.
<img width="243" alt="image" src="https://github.com/saleor/saleor/assets/23282005/3a81ace9-c17e-4d2d-a375-12ad1704ab32">
